docs/CHANGELOG.md

```diff
@@ -12,6 +12,16 @@ The format of this change log follows the advice given at [Keep a CHANGELOG](htt
 ### Added
 - `moodle-plugin-ci install` now provides an option `--db-port` to define a custom database port.
 
+### Changed
+- Updated [gha.dist.yml](https://github.com/moodlehq/moodle-plugin-ci/blob/master/gha.dist.yml) and
+  [.travis.dist.yml](https://github.com/moodlehq/moodle-plugin-ci/blob/master/.travis.dist.yml)
+  (and documentation) to fulfill [Moodle 4.1 new requirements](https://tracker.moodle.org/browse/MDL-71747).
```

Comment:
Typo: "fulfil" is the standard British/Australian spelling (although I admin I don't know which standard you are following already, so please feel free to ignore)

Reply:
Yep, British we are. I really didn't know that both singles existed, amending them now, thanks! Amended!